### PR TITLE
Update QUEST.tsv [Various Lines from 1-300]

### PR DESCRIPTION
**Pull request by ghost**
Just a few tweaks that I couldn't make before #890 was pulled. Reverted Line 90 to a previous translation that I mistook for a very liberal translation but seems to be the actual line.

### CODE REVIEW

File: QUEST.tsv [hunk 1]
```diff
@@ -15,13 +15,13 @@ QUEST_20150317_000014	Goddess Laima
...
 QUEST_20150317_000019	However, knowing of my clairvoyance, the demons would never rest until they saw me gone. {nl}I was forced to hide away, until the time of your appearance.
 QUEST_20150317_000020	I resolved to divide my being and subsist inside these many messages. {nl}I continued to secretly create them, even as the kingdoms flew by over the millennia.
 QUEST_20150317_000021	Savior, these messages are scattered across axes of time. {nl}Once they're all brought together, I will regain my complete form and help against the demons.
 QUEST_20150317_000022	This divine task may be a heavy burden, but it is something you must bear. {nl}Perhaps, in the muddy midst of your trials and tribulations, something may come about.
 QUEST_20150317_000023	Nevertheless, only you and I can save this world.
-QUEST_20150317_000024	I have passed the next revelation to the first Paladin. {nl}I'll be waiting for you at the highest flower garden.
+QUEST_20150317_000024	I've passed the next revelation to the first Paladin. {nl}I'll be waiting for you at the highest flower garden.
```
Comment (Soukyuu) [inline QUEST.tsv:24]: I don't see why this change was necessary.

File: QUEST.tsv [hunk 2]
```diff
@@ -56,14 +56,14 @@ QUEST_20150317_000055	I don't know where to start. {nl}Have you heard anything?
 QUEST_20150317_000056	By royal command, we've come to this forest to investigate some strange events. {nl}Witnesses have reported something with a big face and bony fingers. {nl}What in the world could that be?
 QUEST_20150317_000057	In the past, pilgrims travelled through this forest. {nl}But now, only soldiers gather here due to the monster presence.
 QUEST_20150317_000058	Up until four years ago, the Goddesses often blessed us with signs of their presence. {nl}But now, even our combined prayers don't bring so much as a tremor. {nl}Have they truly abandoned us?
-QUEST_20150317_000059	Vacenin personally goes to each region to deal with things. {nl}He gets uncomfortable when he stays in one place for too long.
-QUEST_20150317_000060	Do you plan on staying or enlisting? {nl}If you're interested, I'll gladly recommend you to the head commanders.
+QUEST_20150317_000059	Vacenin personally goes to each region to deal with things. {nl}He gets uncomfortable when he stays in the city.
+QUEST_20150317_000060	Do you plan on staying or enlisting? {nl}If you're interested, I can see about moving you up.
```
Comment (Soukyuu) [inline QUEST.tsv]: The new version sounds as if they are suggesting to move you up the ranks, instead of recommending your to the ones in charge (as a recruit).

File: QUEST.tsv [hunk 3]
```diff
@@ -87,13 +87,13 @@ QUEST_20150317_000086	The Goddess Vakarine left us shortly before Medis Diena. {
 QUEST_20150317_000087	{memo X}That's right. The strangest one of all was the Goddess Laima. {nl}No one's really sure when she disappeared.
 QUEST_20150317_000088	{memo X}Besides, isn't Laima the Goddess of Fate and Wisdom? {nl}She probably knew all about it.
 QUEST_20150317_000089	Resting Soldier
-QUEST_20150317_000090	I was an adventurer myself just like you, but then I got hit by an arrow…
+QUEST_20150317_000090	I used to be an adventurer like you. Then I took an arrow in the knee…
```
Comment (Soukyuu) [inline QUEST.tsv:90]: I'm guessing that's the "liberal" translation... I wonder if they're allowed to make references to other games just like that.

File: QUEST.tsv (hunk 2 above)
Comment (ghost) [inline QUEST.tsv]: The original is the version of the first translation and the one I changed it to is the translation done by ensata. I'm not quite sure which one is more accurate.

Comment (Soukyuu) [inline QUEST.tsv]: What I'm getting from the KR->JP translation is closer to the first translation, but different:
"Could it be that you are thinking about not staying here and becoming a recruit? If you have such thoughts, I could talk to the upper ranks"

I've asked @SHUcream00 about it on discord.

File: QUEST.tsv (hunk 3 above)
Comment (Soukyuu) [inline QUEST.tsv:90]: According to SHUcream, it's really the Skyrim meme.

File: QUEST.tsv (hunk 2 above)
Comment (Soukyuu) [inline QUEST.tsv]: The original translation is correct, the new one seems like a mistranslation, confirmed by SHUcream on discord. Please revert.

**Comment by ghost** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
Thank you :smile:
